app/src/main/java/com/asma/tasky/feature_management/presentation/edit/EditTitleScreen.kt

```diff
@@ -0,0 +1,57 @@
+package com.asma.tasky.feature_management.presentation.edit
+
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.text.BasicTextField
+import androidx.compose.foundation.text.KeyboardOptions
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.collectAsState
+import androidx.compose.runtime.getValue
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.text.input.ImeAction
+import androidx.compose.ui.text.input.KeyboardCapitalization
+import androidx.compose.ui.text.input.KeyboardType
+import androidx.compose.ui.unit.sp
+import androidx.hilt.navigation.compose.hiltViewModel
+import com.asma.tasky.R
+import com.asma.tasky.core.presentation.ui.theme.SpaceLarge
+import com.asma.tasky.core.presentation.ui.theme.SpaceMedium
+import com.asma.tasky.feature_management.presentation.edit.components.CustomTopBar
+
+@Composable
+fun EditTitleScreen(
+    viewModel: EditViewModel = hiltViewModel(),
+    onClickSave: (String) -> Unit,
+    onBack: () -> (Unit)
+) {
```

Comment:
Instead of defining a separate screen for all types of editing, it could make more sense to just pass the title and default value for the text view to a generic screen, since everything else will remain the same :) 
With your approach if the app gets extended with more string fields in agenda items, you'll have to add more and more screens for each type